testing/test-cases.d/general.basic.passthrough/test.sh

```diff
@@ -18,7 +18,8 @@ echo -n "LINE: ${LINENO} " && syslog_waitfor 10 'ods-signerd: .*\[STATS\] exampl
 echo -n "LINE: ${LINENO} " && test -f "$INSTALL_ROOT/var/opendnssec/signed/example.com" &&
 
 ## test absence of signatures
-echo -n "LINE: ${LINENO} " && grep -vq RRSIG "$INSTALL_ROOT/var/opendnssec/signed/example.com" &&
+echo -n "LINE: ${LINENO} " && grep -q RRSIG "$INSTALL_ROOT/var/opendnssec/signed/example.com" &&
+echo -n "LINE: ${LINENO} " && if grep -q RRSIG "$INSTALL_ROOT/var/opendnssec/signed/example.com" ; then false ; fi &&
```

Comment:
I see why the original is wrong. But here you are doing  "RRSIG && !RRSIG" making the test fail always?